// bippy/dist/source.cjs
/**
 * @license bippy
 *
 * Copyright (c) Aiden Bai
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */
const require_source = require('./source-CTbJ4jWB.cjs');
require('./rdt-hook-CR3KoPkF.cjs');
require('./src-BIDabo17.cjs');

exports.describeBuiltInComponentFrame = require_source.describeBuiltInComponentFrame;
exports.describeClassComponentFrame = require_source.describeClassComponentFrame;
exports.describeDebugInfoFrame = require_source.describeDebugInfoFrame;
exports.describeFiber = require_source.describeFiber;
exports.describeFunctionComponentFrame = require_source.describeFunctionComponentFrame;
exports.describeNativeComponentFrame = require_source.describeNativeComponentFrame;
exports.disableLogs = require_source.disableLogs;
exports.formatOwnerStack = require_source.formatOwnerStack;
exports.getCurrentDispatcher = require_source.getCurrentDispatcher;
exports.getFiberSource = require_source.getFiberSource;
exports.getFiberStackFrame = require_source.getFiberStackFrame;
exports.getFiberStackTrace = require_source.getFiberStackTrace;
exports.getOwnerStack = require_source.getOwnerStack;
exports.getRemovedFileProtocolPath = require_source.getRemovedFileProtocolPath;
exports.getSourceMap = require_source.getSourceMap;
exports.isCapitalized = require_source.isCapitalized;
exports.parseStackFrame = require_source.parseStackFrame;
exports.reenableLogs = require_source.reenableLogs;
exports.setCurrentDispatcher = require_source.setCurrentDispatcher;
